### PR TITLE
add tide configuration for assisted-installer-deployment

### PR DESCRIPTION
**Pull request by osherdp**
/cc  @droslean
/hold

### CODE REVIEW

File: core-services/prow/02_config/openshift-assisted/assisted-installer-deployment/_prowconfig.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      repos:
+        assisted-installer-deployment:
+          branches:
+            master:
+              protect: true
+          protect: false
+tide:
+  merge_method:
+    openshift-assisted/assisted-installer-deployment: squash
+  queries:
```
Comment (droslean) [inline core-services/prow/02_config/openshift-assisted/assisted-installer-deployment/_prowconfig.yaml:13]: you should include or exclude in which branches you need this configuration to be applied.

Comment (osherdp) [reply core-services/prow/02_config/openshift-assisted/assisted-installer-deployment/_prowconfig.yaml:13]: so changed to have those validations on all branches
is that right?

Comment (droslean) [reply core-services/prow/02_config/openshift-assisted/assisted-installer-deployment/_prowconfig.yaml:13]: correction: It shouldn't be mandatory though. Lets merge this.

**Review by droslean — APPROVED** [review]

**Review by droslean — APPROVED** [review]

**Pull request merged**